--- src/st/config.def.h	2018-03-31 15:14:20.188541121 -0500
+++ src/st/config.h	2018-08-27 22:20:43.138640969 -0500
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "tewi:antialias=true";
 static int borderpx = 2;
 
 /*
@@ -82,40 +82,38 @@
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-};
+    "#1d1f21", /* base00 */
+    "#cc342b", /* base08 */
+    "#198844", /* base0B */
+    "#fba922", /* base0A */
+    "#3971ed", /* base0D */
+    "#a36ac7", /* base0E */
+    "#3971ed", /* base0C */
+    "#c5c8c6", /* base05 */
+
+    "#969896", /* base03 */
+    "#cc342b", /* base08 */
+    "#198844", /* base0B */
+    "#fba922", /* base0A */
+    "#3971ed", /* base0D */
+    "#a36ac7", /* base0E */
+    "#3971ed", /* base0C */
+    "#ffffff", /* base07 */
+    
+    "#f96a38", /* base09 */
+    "#3971ed", /* base0F */
+    "#282a2e", /* base01 */
+    "#373b41", /* base02 */
+    "#b4b7b4", /* base04 */
+    "#e0e0e0", /* base06 */
 
+    [255] = 0,
+
+    "#c5c8c6", /* base05 */
+    "#1d1f21", /* base00 */
+};
 
-/*
- * Default colors (colorname index)
- * foreground, background, cursor, reverse cursor
- */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
 static unsigned int defaultcs = 256;
